pytest.ini

```diff
@@ -6,6 +6,6 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 [pytest]
-pep8ignore = docs/conf.py ALL
-addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=invenio_rdm_pure --cov-report=term-missing
-testpaths = docs tests invenio_rdm_pure
+addopts = --isort --pydocstyle --pycodestyle --doctest-glob="*.rst" --doctest-modules --cov=invenio_rdm_pure --cov-report=term-missing
+testpaths = tests invenio_rdm_pure
+live_server_scope = module
```

Comment:
please add a newline add the end of the file

Reply:
Why a new line is necessary?

Reply:
it is best described in this link: https://stackoverflow.com/a/729795